src/tests/multihost/ipa/conftest.py

```diff
@@ -97,6 +138,32 @@ def remove_ipa_users():
     request.addfinalizer(remove_ipa_users)
 
 
+@pytest.fixture(scope="class")
+def default_ipa_groups(session_multihost, request):
+    """ Create IPA Groups ipa-group0 to ipa-group9 """
+    kinit_admin = 'kinit admin'
+    session_multihost.master[0].run_command(kinit_admin,
+                                            stdin_text='Secret123',
+                                            raiseonerr=False)
+    ipa_grp_gid_start = 342156780
+    ipa_grp_count = 10
+    for i in range(ipa_grp_count):
+        gid = ipa_grp_gid_start + i
+        group_add = 'ipa group-add ipa-group%d --desc="IPA group%d" --gid %d'\
+                    % (i, i, gid)
+
+        cmd = session_multihost.master[0].run_command(group_add,
+                                                      raiseonerr=False)
+        assert cmd.returncode == 0
+
+    def remove_ipa_groups():
+        """ Remove ipa Groups ipa-group0 to ipa-group9 """
+        for i in range(10):
+            cmd = 'ipa group-del ipa-group%d' % i
+            session_multihost.master[0].run_command(cmd)
+    request.addfinalizer(remove_ipa_groups)
+
```

Comment:
Please add here so the test knows what gids to use:
    return ipa_grp_gid_start, ipa_grp_count